src/pages/Admin/Room/Edit/index.ts

```diff
@@ -1 +1 @@
-export { default } from "./RoomEdit";
+export { default } from "./RoomEditModal";
```

Comment:
Prefer explicitly named exports instead of `default`

```suggestion
export { RoomEditModal } from "./RoomEditModal";
```

(this is also going to require changing how it is imported in the files that use this, pretty much just by wrapping it in `{` and `}` (you'll see other examples of how this is done)